src/module.js

```diff
@@ -1,10 +1,15 @@
 (function(exports) {
   "use strict";
 
-// Define the `firebase` module under which all AngularFire
-// services will live.
-  angular.module("firebase", [])
-    //todo use $window
-    .value("Firebase", exports.Firebase);
+  angular.module("firebase.utils", []);
+  angular.module("firebase.config", []);
+  angular.module("firebase.auth", ["firebase.utils"]);
+  angular.module("firebase.database", ["firebase.utils"]);
 
-})(window);
\ No newline at end of file
+  // Define the `firebase` module under which all AngularFire
+  // services will live.
+  angular.module("firebase", ["firebase.utils", "firebase.config", "firebase.auth", "firebase.database"])
+    //TODO: use $window
+    .value("Firebase", exports.firebase)
```

Comment:
Makes sense. Let's at least add a `TODO` comment here to remove this line when the `3.0.0` release happens.